docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -118,6 +209,31 @@ Alternatively, you can use the `kubectl autoscale` command to accomplish the sam
 kubectl autoscale rs frontend
 ```
 
+## Alternatives to ReplicaSet
```

Comment:
Is there any reason why StatefulSet is left out in this section?

Reply:
StatefulSet is for stateful applications that have very different requirements than those of stateless applications, which ReplicaSet and the alternatives can potentially handle.